File: JavaScript/leetcode75/src/test/Problem70_ImplementTriePrefixTree.test.js
const Trie = require("../main/Problem70_ImplementTriePrefixTree");

test("should work as trie", () => {
  let trie = new Trie();
  trie.insert("apples");
  expect(trie.search("apples")).toBeTruthy();
  expect(trie.startsWith("app")).toBeTruthy();
});

File: Java/leetcode75/src/test/java/org/phoenix/leetcode/challenges/Problem70_ImplementTriePrefixTreeTest.java
package org.phoenix.leetcode.challenges;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertTrue;

class Problem70_ImplementTriePrefixTreeTest {

    private final Problem70_ImplementTriePrefixTree test = new Problem70_ImplementTriePrefixTree();


    @Test
    void search() {
        test.insert("apples");
        assertTrue(test.search("apples"));
        assertTrue(test.startsWith("app"));
    }

    @Test
    void startsWith() {
    }
}